packages/template-retail-react-app/app/components/order-summary/index.jsx

```diff
@@ -20,7 +20,7 @@ import CartItemVariantPrice from '../item-variant/item-price'
 import PromoPopover from '../promo-popover'
 
 const CartItems = ({basket}) => {
-    basket = basket || useBasket()
+    basket = basket || useCurrentBasket().basket
```

Comment:
```suggestion
    basket ||= useCurrentBasket().basket
```
Fun fact, `||=` is a thing!